src/main.cpp

```diff
@@ -12,15 +12,15 @@ void signal_handler(int signal) { std::exit(0); }
 
 int main(int argc, char *argv[]) {
 	uint16_t port{8080};
-	std::string data_path{"."};
+	std::string data_path{"./"};
```

Comment:
I think it was working before without the slash.. maybe i messed something up deeper inside krabby

Reply:
you are adding slash to pah read from option, so default must also contain slash

Reply:
Environment simply adds (+) filename to it, so either data_path must be empty or "./".

Reply:
I thought that if i'm adding the slash somewhere later it was going to add it to the default "." as well. Maybe i was wrong. In any case the correct solution here would be to have it work for any path with or without a trailing slash since users can use arbitrary paths anyway.. I will fix that later sometime